zio-http/jvm/src/main/scala/zio/http/netty/server/NettyResponseEncoder.scala

```diff
@@ -14,51 +14,55 @@
  * limitations under the License.
  */
 
-package zio.http.netty
+package zio.http.netty.server
```

Comment:
Moved this under the `server` package since these methods are _very_ server specific